### ISSUE TITLE
NVDA doesn't read the "Windows protection" dialogs

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3150]
**Reported by k_kolev1985 on 2013-04-12 15:55**
When installing an application witch installs drivers (e.g. VirtualBox with default installation parameters), a dialog may appear asking you if you want to install drivers from unknown/untrusted developer. NVDA doesn't read the content of that dialog (neither automatically, nor by request with NVDA+B) - only it's title, witch I think in english is "Windows protection" or "Windows security". There are a few focusable controls in that dialog (I think a check-box, 2 buttons, and a link), but when moving around with TAB between them, NVDA doesn't read their labels as well - it stays silent.

**Comment by ehollig** [comment]
@k_kolev1985, do you have more information about this issue that you could share so that I, and others can replicate it? Also, an error log demonstrating the issue also may be useful.

**Comment by k-kolev1985** [comment]
Actually, not much really. I haven't encountered such a dialog in a long time and I've stopped using VirtualBox a long time ago. But I was able to reproduce the issue again recently, while installing ViewScan from [https://www.hamrick.com](url). It installs a scanner driver and the same security prompt appears. If I remember correctly, I had to restart NVDA to make it read that dialog prompt. The only specific setting to my system that I can think of is that I'm using the "High-Contrast Black" theme in Windows, but would that matter (?) - I don't know.

**Comment by lukaszgo1** [comment]
I am able to interact with these dialogs both on Windows 7 and on Windows 10. @k-kolev1985 Are you still having this issue? If not i vote to close it.

**Comment by k-kolev1985** [comment]
@lukaszgo1 Try it with high-contrast enabled in Windows and see if you'll have the same results. I may try it with high-contrast disabled these days if I get a chance and see if that makes any difference for me.

**Comment by lukaszgo1** [comment]
@k-kolev1985 I've tried with high-contrast enabled on Windows 10 1903 and Virtual Box installer and these dialog are accessible for me - they even automatically gained focus.

**Comment by bdorer** [comment]
Hi @k-kolev1985 and @lukaszgo1 did anyboddy do further tests? If not, 
please do and reply.

Thanks

**Comment by Adriani90** [comment]
It might also depend on which level you have set on UAC. Or if you have the right privileges on the system.
Anyway, it seems we do not have real chances to reproduce this issue and it is really difficult to investigate further. If you still see this issue and it is not solved after starting NVDA as administrator or after changing UAC level, please comment on it and we can reopen.
Try also running the COM registration fixing tool.
